site/website/package.json

```diff
@@ -31,6 +31,8 @@
     "wait-on": "^4.0.0"
   },
   "dependencies": {
-    "concurrently": "^5.1.0"
+    "concurrently": "^5.1.0",
+    "@material-ui/core": "^4.9.10",
```

Comment:
Do we really need Material UI just for Card component ?

Reply:
I removed the unnecessary dependencies.